handbook/ops/bizops/customer_data_policy.md

```diff
@@ -1,15 +1,53 @@
 # Customer data policy
+
 How Sourcegraph handles customer information.
 
+Product data
+| Source/platform | Sourcegraph Cloud | Self-hosted | How long is this data kept for? | Who has access? |
+|------------------------------------------------------------------|--------------------|--------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|
+| Ping telemetry | | :heavy_check_mark: | Delete data for any instance that’s been offline for over 3 years. All active instance data are kept | All Sourcegraph teammates |
+| Google Cloud Platform data pipeline (Buckets, Dataflow, Pub/Sub) | :heavy_check_mark: | :heavy_check_mark: | Pub/Sub and Dataflow don’t hold data (only manage the data pipelines). GCP buckets follow the same rules as the data they backup (i.e. Sourcegraph.com data backed up in buckets is kept for a year) | Analytics and engineering teams |
```

Comment:
What does the Self-hosted check-mark for GCP data pipeline mean? On-prem customers regularly ask us what Customer data we (Sourcegraph personnel) have access to. We generally reply that the only data we get is Usage Data/Ping telemetry. Should we incorporate GCP data into our responses? If so, can you elaborate on the types of data Sourcegraph personnel can access?